sql/hive/src/main/scala/org/apache/spark/sql/hive/hiveUDFs.scala

```diff
@@ -586,47 +524,78 @@ private[hive] case class HiveGenericUDTF(
 
 private[hive] case class HiveUDAFFunction(
```

Comment:
It will be better to explain that if a query has a HiveUDAF, then we will not do partial aggregation.